From c5b91c2e7cc259396fb19dc03fc50d3150d120df Mon Sep 17 00:00:00 2001
From: Lei wang <quic_leiwan@quicinc.com>
Date: Fri, 2 Feb 2024 15:42:01 +0800
Subject: [PATCH] centos-5.14: Fix redhad env

Specify the right path for makefile.rhelver in yocto build
this will help kernel module identify rhelver.

Upstream-Status: Inappropriate

Change-Id: Ic8473bba00b88eb1ecd9acfcf28f4a979abfd716
Signed-off-by: Lei wang <quic_leiwan@quicinc.com>
---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c1f7c8596907..db930c51af24 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,9 @@ __all:
 
 # Set RHEL variables
 # Use this spot to avoid future merge conflicts
-include Makefile.rhelver
+this_makefile_rhel := $(lastword $(MAKEFILE_LIST))
+abs_srctree_rhel := $(realpath $(dir $(this_makefile_rhel)))
+include $(abs_srctree_rhel)/Makefile.rhelver
 
 # We are using a recursive build, so we need to do a little thinking
 # to get the ordering right.
-- 
2.17.1

